package.json

```diff
@@ -49,15 +49,14 @@
     "test-browser-drive": "NODE_ENV=test node ./test/browser-drive.js debug",
     "test-node-debug": "NODE_ENV=test node -r ./babel-register.js ./test/node.js",
     "test-browser-debug": "NODE_ENV=test node -r ./babel-register.js ./test/setup-browser-env.js ./test/js-dom.js",
-    "test-jest": "jest",
+    "test-jest": "jest --coverage",
     "test-tape": "yarn test-node && yarn test-browser",
     "test": "yarn test-jest && yarn test-tape",
-    "cover-tape": "nyc --reporter=json --report-dir=tape-coverage --reporter=html yarn test-tape",
-    "cover-jest": "yarn jest --coverage --reporter=html --watchAll=false",
-    "cover-copy-tape": "yarn nyc merge tape-coverage .nyc_output/coverage.json",
-    "cover-copy-jest": "yarn nyc merge jest-coverage .nyc_output/coverage.json",
-    "cover-copy": "rm -rf .nyc_output && mkdir .nyc_output && yarn cover-copy-tape && yarn cover-copy-jest",
-    "cover": "yarn cover-jest && yarn cover-tape && yarn cover-copy && nyc report --reporter=json --reporter=html --reporter=lcov",
+    "cover": "yarn cover-tape && yarn cover-jest && yarn cover-merge && yarn cover-report",
+    "cover-tape": "nyc --reporter=json --report-dir=tape-coverage --reporter=lcov yarn test-tape",
+    "cover-jest": "yarn jest --coverage --reporter=lcov --watchAll=false",
+    "cover-merge": "istanbul-merge --out=coverage/coverage-all.json tape-coverage/coverage-final.json jest-coverage/coverage-final.json",
```

Comment:
Nice, didn't know merging reports was possible. Could help us introduce jest in other frameworks!

Reply:
yes, jest generates its own coverage report in the same format as nyc by running `jest --coverage`, so using nyc to run jest doesn't work. But with istanbul-merge, we can merge the report generated by nyc and jest into one.